docs/guide/imposter.rst

```diff
@@ -0,0 +1,54 @@
+Conquering Imposter Syndrome
+============================
+You are a technical writer. At some point in your career, you may face the notion that you are "just" a technical writer. That notion might originate from the perception of your more technical coworkers, or your own self-doubt. 
+
+Some common themes emerge in many teams: anyone can write, but writers don't code, so they can't understand technical content. There might even be a notion that the docs should be written by engineers, but they either don't want to, or don't have the time. The actual product or code is going to trump its accompanying documentation when push comes to shove. It might seem logical to accept that writing is going to take a backseat and garner less esteem. 
```

Comment:
```suggestion
Some common themes emerge in many teams: anyone can write, but many writers don't code, so they can't understand technical content. There might even be a notion that the docs should be written by engineers, but they either don't want to, or don't have the time. The actual product or code is going to trump its accompanying documentation when push comes to shove. It might seem logical to accept that writing is going to take a backseat and garner less esteem. 
```